engines/real-time-ensembler/pyfunc_ensembler_runner/handler.py

```diff
@@ -0,0 +1,31 @@
+import tornado.web
+import orjson
+import json
+
+from http import HTTPStatus
+from typing import Any, Dict
+from pyfunc_ensembler_runner.ensembler_runner import PyFuncEnsemblerRunner
+
+
+class EnsemblerHandler(tornado.web.RequestHandler):
+    def initialize(self, ensembler: PyFuncEnsemblerRunner):
+        self.ensembler = ensembler
+
+    def post(self):
+        request = EnsemblerHandler.validate_request(self.request)
+        response = self.ensembler.predict(request)
+
+        response_json = orjson.dumps(response)
+        self.write(response_json)
+        self.set_header("Content-Type", "application/json; charset=UTF-8")
+
+    @staticmethod
+    def validate_request(request: Any) -> Dict[str, Any]:
+        try:
+            body = orjson.loads(request.body)
```

Comment:
Is this the same parser, that @gojek/merlin is using in their PyFunc model server?

Reply:
Yep that's right